dev-client/src/screens/HomeScreen.tsx

```diff
@@ -275,50 +280,67 @@ const LandPKSInfo = () => {
 
   return (
     <BottomSheetScrollView>
-      <Column space={3} pb="65%" px={5} mt="48px">
+      <Column space={3} pb="65%" pt={5} px={5} mt="48px">
         <Heading w="full" textAlign="center">
           {t('home.info.title')}
         </Heading>
         <Image
           source={require('terraso-mobile-client/assets/landpks_intro_image.png')}
           w="100%"
-          h="30%"
+          h="25%"
           resizeMode="contain"
           alt={t('home.info.intro_image_alt')}
         />
-        <Text>
-          <Text bold>{t('home.info.description.lead')} </Text>
-          {t('home.info.description.body')}
-        </Text>
-        <Text alignItems="center">
-          <Text bold>{t('home.info.location.lead')} </Text>
-          <Trans
-            i18nKey="home.info.location.body"
-            components={{
-              icon: (
-                <Icon
-                  name="my-location"
-                  color="action.active"
-                  position="relative"
-                />
-              ),
-            }}
-          />
-        </Text>
-        <Text>
-          <Text bold>{t('home.info.search.lead')} </Text>
-          {t('home.info.search.body')}
+        <Text variant="body1">
+          <Trans i18nKey="home.info.description">
+            <Text bold>first</Text>
+            <Text>second</Text>
+            <Text bold>third</Text>
+          </Trans>
         </Text>
-        <Text>
-          <Text bold>{t('home.info.learn_more.lead')} </Text>
+        <HStack key={1}>
+          <Text variant="body1" mr={2}>
+            1
+          </Text>
+          <Text variant="body1" mr={2}>
+            {t('home.info.list1')}
+          </Text>
+        </HStack>
+        <HStack key={2}>
+          <Text variant="body1" mr={2}>
+            2
+          </Text>
+          <Text variant="body1">
+            <Trans
+              i18nKey="home.info.list2"
+              components={{
+                icon: <LocationIcon />,
+              }}
+            />
+          </Text>
+        </HStack>
+        <HStack key={3}>
+          <Text variant="body1" mr={2}>
+            3
+          </Text>
+          <Text variant="body1" mr={2}>
+            {t('home.info.list3')}
+          </Text>
+        </HStack>
```

Comment:
suggestion:
```suggestion
    <FlatList 
        data={["home.info.list1", "home.info.list2", "home.info.list2"]} 
        renderItem={({index, item}) => (
        <HStack>
            <Text variant="body1" mr={2}>{index}</Text>
            <Text variant="body1" mr={2}>{t(item)}</Text>
        </HStack>)} 
        keyExtractor={(item) => item} 
    />
```

(I didn't test it out, so there may be some syntax issues).